src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -22,44 +33,49 @@
 "_comment": "STATIONS",
 
 	"block.mmorpg.teleporter": "Картографическое устройство",
+	"block.mmorpg.station.cooking": "Станция готовки",
+	"block.mmorpg.station.salving": "Станция утилизации",
+	"block.mmorpg.station.gear_crafting": "Станция снаряжения",
+	"block.mmorpg.station.alchemy": "Алихимическая станция",
```

Приводим к единому формату
```suggestion
	"block.mmorpg.station.alchemy": "Станция алхимика",
```